src/pages/atomicdex/api/v20-dev/lightning/channels/index.mdx

```diff
@@ -0,0 +1,932 @@
+```json {{ mm2MethodDecorate : true }}
+{
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "method": "lightning::channels::list_open_channels_by_filter",
+    "params": {
+        "coin": "tBTC-lightning",
+        "filter": {
+            "is_outbound": false
+        }
+    },
+    "id": 55
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+
+#### Response
+
+```json
+{
+	"mmrpc": "2.0",
+	"result": {
+		"open_channels": [{
+			"uuid": "f86dbfae-898a-4f49-8149-58d9dfa095a5",
+			"channel_id": "143cd1fa265c4bed860ce81f369d5c4ee8ef80c5c91872176f524348c2c1fff4",
+			"counterparty_node_id": "03cf982b1c16f7d3561d8bb17f7cf30057389d228756bce517c0f3cc111b38ecd0",
+			"funding_tx": "f4ffc1c24843526f177218c9c580efe84e5c9d361fe80c86ed4b5c26fad13c14",
+			"funding_tx_output_index": 0,
+			"funding_tx_value_sats": 40000,
+			"is_outbound": false,
+			"balance_msat": 5211,
+			"outbound_capacity_msat": 0,
+			"inbound_capacity_msat": 38994789,
+			"current_confirmations": 215,
+			"required_confirmations": 3,
+			"is_ready": true,
+			"is_usable": false,
+			"is_public": false
+		}],
+		"limit": 10,
+		"skipped": 0,
+		"total": 1,
+		"total_pages": 1,
+		"paging_options": {
+			"PageNumber": 1
+		}
+	},
+	"id": 55
+}
+```
+</CollapsibleSection>
+
+
+#### 📌 Example with `filter` for a specific `node_id`
+
+#### Command
+
+<CodeGroup title="" tag="POST" label="list_open_channels_by_filter">
+```json {{ mm2MethodDecorate : true }}
+{
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "method": "lightning::channels::list_open_channels_by_filter",
+    "params": {
+        "coin": "tBTC-lightning",
+        "filter": {
+            "counterparty_node_id": "02eb0b178576857b6990ba57d56aa08f651a05a8098496004f42df5e7440b0a9c1"
+        }
+    },
+    "id": 55
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+
+#### Response
+
+```json
+{
+	"mmrpc": "2.0",
+	"result": {
+		"open_channels": [{
+			"uuid": "adde8899-ba11-435a-9433-d180e2f5af6b",
+			"channel_id": "4cde288dd1ec1691b51cac8890a867a58fcfc98670e32ed7a112402b819a01f1",
+			"counterparty_node_id": "02312627fdf07fbdd7e5ddb136611bdde9b00d26821d14d94891395452f67af248",
+			"funding_tx": "f1019a812b4012a1d72ee37086c9cf8fa567a89088ac1cb59116ecd18d28de4c",
+			"funding_tx_output_index": 0,
+			"funding_tx_value_sats": 40000,
+			"is_outbound": true,
+			"balance_msat": 38998197,
+			"outbound_capacity_msat": 38598197,
+			"inbound_capacity_msat": 1803,
+			"current_confirmations": 215,
+			"required_confirmations": 3,
+			"is_ready": true,
+			"is_usable": true,
+			"is_public": false
+		}],
+		"limit": 10,
+		"skipped": 0,
+		"total": 1,
+		"total_pages": 1,
+		"paging_options": {
+			"PageNumber": 1
+		}
+	},
+	"id": 55
+}
+```
+</CollapsibleSection>
+
+
+#### 📌 Example with `filter` for a mode with between 100000 and 500000 satoshi funding value
```

Comment:
Type in `mode`, should be `node`. It's in other places too

Reply:
fixed thanks